README.md

```diff
@@ -1,3 +1,84 @@
+Steps:
+
+1. Clone the Git repository:
+```
+git clone https://github.com/nationalarchives/ctd-omega-services-prototype.git
+```
+2. Switch to the newly created directory:
+```
+cd ./ctd-omega-services
+```
+3. For setting up ElasticMQ, install and start Docker Desktop according to the operating system from here: https://docs.docker.com/desktop/
+
+    For Linux, Install the docker-engine from https://docs.docker.com/engine/install/#server and docker-compose from https://docs.docker.com/compose/install/ respectively.
+
+4. From the `ctd-omega-services` project root and run `docker-compose.yml` in console as below:
+```
+docker-compose up -d
+```
+5. Run the `docker` command to verify the docker instance is running for ElasticMQ as below:
+```
+docker ps -a
+```
+6. Go to http://localhost:9325/ in a web browser and you should see the message queues configured in `custom.conf`, e.g. `request-general` and `omega-editorial-web-application-instance-1`
+
+7. To run the Pan-Archival Catalogue Services locally you can use ` sbt run`. Note that the queue names configured in the `application.conf` file used by application must match those in the `custom.conf` file used by ElasticMQ.
+
+8. Your message service is now running and ready to respond to requests.
+
+## Packaging
+This project makes use of the [sbt-native-packager](https://www.scala-sbt.org/sbt-native-packager/index.html) plugin to create distribution packages of the application.
+
+In order to create a zip package, which includes the application bundled as a jar file, together with its dependent jar files and also scripts to start the application, run the following sbt command:
+```
+sbt universal:packageBin
+```
+The zip file will be created in the `./target/universal` directory of the project.
+It is also possible to preview the distribution using by running:
+```
+sbt stage
+```
+This creates the package files in the `./target/universal/stage` directory from where they can be examined and tested.
+
+It is also possible to build a local Docker image of the application with the following command:
+```
+sbt docker:publishLocal
+```
+You can verify the existence of the Docker image by running the following from the command line:
+```  
+docker image ls
+```
+It is recommended to [read more about the Docker Plugin](https://www.scala-sbt.org/sbt-native-packager/formats/docker.html) if you wish yo use this feature. 
+## Running the packaged application
+When the application has been packaged as described above, the following directory structure will be created in the distribution:
+```
+│
+└─bin
+│   ctd-omega-services
+│   ctd-omega-services.bat
+└─lib
+uk.gov.nationalarchives.ctd-omega-service.jar
+...
+```
+The `bin` folder contains the scripts to start the application from the command line and the `lib` folder contains the application jar file and other dependencies.
+
+Run the application from the command line using the `ctd-omega-services` script or `ctd-omega-service.bat` Windows batch file.
```

Comment:
Or with `sbt run`. 

Would also be good to have some dev-centric info like how to run the tests (both unit and integration).

Reply:
Added description of how to run unit tests. Integration tests will be added in [PACT-814](https://national-archives.atlassian.net/browse/PACT-814).

Reply:
Confirmed.